en/How to/Import data.md

```diff
@@ -32,21 +32,21 @@ If you have data in Roam, it's easy to export and play around with it in Obsidia
 > 4. Click on the vault icon (the tooltip should say "Open another vault") on the bottom left corner in the app, and select "Open folder as vault".
 > Pick the folder you just unzipped to.
 
-> 5. Use our [[markdown format converter]] to convert Roam Research's Markdown format into Obsidian's format.
+> 5. Use our [[vi/Plugin/Markdown format converter]] to convert Roam Research's Markdown format into Obsidian's format.
```

Comment:
Did you open the `en` folder while working? That would prevent you from linking to pages under other folders

Reply:
```suggestion
> 5. Use our [[Format converter]] to convert Roam Research's Markdown format into Obsidian's format.
```